### PR TITLE
changed a comment in KanesMethod at around line 340

### PR DESCRIPTION
**Pull request by Peter230655**
#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by Peter230655** [comment]
I made my commit before I updated my branch, this may have caused failure in test / Numpy/scipy nightly?
I will correct and start a new pull request.

**Comment by moorepants** [comment]
In SymPy there are require test runs and ones that aren't required. The numpy nightly is not required.

**Comment by Peter230655** [comment]
> In SymPy there are require test runs and ones that aren't required. The numpy nightly is not required.

I did not look! 
What about that name thing? Is this fatal?
I corrected the description as required - I hope.

**Comment by moorepants** [comment]
For your first PR in SymPy you have to add you name to the authors list.

**Comment by moorepants** [comment]
See: https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#mailmap-instructions

**Comment by Peter230655** [comment]
> See: https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#mailmap-instructions

I looked there, but I cannot lokate this .mailmap file, where, presumably I should enter:
Peter Stahlecker <peter.stahlecker@gmail.com>        Peter230655 <peter.stahlecker@gmail.com>

I opened mailmap_check.py in VSC and ran it with the Git Bash terminal I installed there:

image

**Comment by moorepants** [comment]
https://github.com/sympy/sympy/blob/master/.mailmap

**Comment by Peter230655** [comment]
> https://github.com/sympy/sympy/blob/master/.mailmap

I found it there. But how do I add what is needed?
Would
Peter Stahlecker <peter.stahlecker@gmail.com>     Peter230655 <peter.stahlecker@gmail.com>
be correct?

**Comment by moorepants** [comment]
You edit it in this branch.

File: .mailmap [hunk 1]
```diff
@@ -1674,3 +1674,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
 袁野 (Yuan Ye) <yuanyelele@tutanota.com>
+Peter Stahlecker <peter.stahlecker@gmail.com>   Peter230655 <peter.stahlecker@gmail.com>
```
Comment (moorepants) [inline .mailmap:1677]:
```suggestion
Peter Stahlecker <peter.stahlecker@gmail.com>   Peter230655 <83544146+Peter230655@users.noreply.github.com>
```

Comment (Peter230655) [reply .mailmap:1677]: > 

This change is to be made in the .mailmap  ?

Comment (moorepants) [reply .mailmap:1677]: yes

Comment (Peter230655) [reply .mailmap]: I think worked unless I made a typo.

image

this is what I got with cat .mailmap

image

Is a PR required, or not?

**Thanks for all the help!!**
Before I turn 75, I will know it for sure! :-)

**Comment by moorepants** [comment]
Your commits include two name-email combinations:
```
commit a1c0aefe3cb297047776ca135b71a780db3f1bbe (HEAD -> pr-26853)
Author: Peter Stahlecker <peter.stahlecker@gmail.com>
Date:   Fri Jul 26 09:16:46 2024 +0200

    entered Peter Stahlecker into .mailmap

commit 609b061eba98148d0d2d7edf2ae2c6e57bf6decc
Merge: 4a5f9dd118 ea05c48e8f
Author: Peter Stahlecker <peter.stahlecker@gmail.com>
Date:   Thu Jul 25 18:41:23 2024 +0200

    Merge branch 'KanesMethodComplex' of github.com:Peter230655/sympy-pst into KanesMethodComplex

commit 4a5f9dd118458becbcaa857a7d55717717f57e94
Author: Peter Stahlecker <peter.stahlecker@gmail.com>
Date:   Thu Jul 25 16:47:52 2024 +0200

    added Peter Stahlecker to .mailmap

commit ea05c48e8fae0d1a161792598ab7d6029f315e45
Merge: 60f446fa18 958cc95aeb
Author: Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
Date:   Wed Jul 24 18:28:49 2024 +0200

    Merge branch 'sympy:master' into KanesMethodComplex

commit 60f446fa18dcd9803108f95d8a7ac286ab0efb48
Author: Peter Stahlecker <peter.stahlecker@gmail.com>
Date:   Wed Jul 24 18:25:08 2024 +0200

    typo around line 340 time-derivatives of generalized coordinates
```

I think that means you need two lines in `.mailmap` like:

```
Peter Stahlecker <peter.stahlecker@gmail.com> Peter Stahlecker <peter.stahlecker@gmail.com>
Peter Stahlecker <peter.stahlecker@gmail.com> Peter230655 <83544146+Peter@users.noreply.github.com>
```

**Comment by Peter230655** [comment]
> Your commits include two name-email combinations:
> 
> ```
> commit a1c0aefe3cb297047776ca135b71a780db3f1bbe (HEAD -> pr-26853)
> Author: Peter Stahlecker <peter.stahlecker@gmail.com>
> Date:   Fri Jul 26 09:16:46 2024 +0200
> 
>     entered Peter Stahlecker into .mailmap
> 
> commit 609b061eba98148d0d2d7edf2ae2c6e57bf6decc
> Merge: 4a5f9dd118 ea05c48e8f
> Author: Peter Stahlecker <peter.stahlecker@gmail.com>
> Date:   Thu Jul 25 18:41:23 2024 +0200
> 
>     Merge branch 'KanesMethodComplex' of github.com:Peter230655/sympy-pst into KanesMethodComplex
> 
> commit 4a5f9dd118458becbcaa857a7d55717717f57e94
> Author: Peter Stahlecker <peter.stahlecker@gmail.com>
> Date:   Thu Jul 25 16:47:52 2024 +0200
> 
>     added Peter Stahlecker to .mailmap
> 
> commit ea05c48e8fae0d1a161792598ab7d6029f315e45
> Merge: 60f446fa18 958cc95aeb
> Author: Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
> Date:   Wed Jul 24 18:28:49 2024 +0200
> 
>     Merge branch 'sympy:master' into KanesMethodComplex
> 
> commit 60f446fa18dcd9803108f95d8a7ac286ab0efb48
> Author: Peter Stahlecker <peter.stahlecker@gmail.com>
> Date:   Wed Jul 24 18:25:08 2024 +0200
> 
>     typo around line 340 time-derivatives of generalized coordinates
> ```
> 
> I think that means you need two lines in `.mailmap` like:
> 
> ```
> Peter Stahlecker <peter.stahlecker@gmail.com> Peter Stahlecker <peter.stahlecker@gmail.com>
> Peter Stahlecker <peter.stahlecker@gmail.com> Peter230655 <83544146+Peter@users.noreply.github.com>
> ```

No problem, now that I know how to do it. :-)

File: .mailmap [hunk 2]
```diff
@@ -1674,3 +1674,5 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
 袁野 (Yuan Ye) <yuanyelele@tutanota.com>
+Peter Stahlecker <peter.stahlecker@gmail.com>   Peter230655 <83544146+Peter@users.noreply.github.com>
+Peter Stahlecker <peter.stahlecker@gmail.com>  Peter Stahlecker <peter.stahlecker@gmail.com
```
Comment (oscarbenjamin) [inline .mailmap]: The two names reported by the mailmap check script are:
```
	Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
	Peter Stahlecker <peter.stahlecker@gmail.com>
```
What needs to be added is just one line like:
```
Peter Stahlecker <peter.stahlecker@gmail.com> Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
```
After adding that line you need to run the `bin/mailmap_check.py` script twice. The first run will sort the file into alphabetical order. The second will confirm that the .mailmap file is correct. Then commit and push the changes.

Currently the script shows that your name will be added twice to the AUTHORS file which is not what we want:
```
The following authors will be added to the AUTHORS file at the time of
the next SymPy release.

	Andrew Mosson <amosson@yahoo.com>
	Edward Z. Yang <ezyang@mit.edu>
	Gonzalo Tornaría <tornaria@cmat.edu.uy>
	James Titus <titusjames299@gmail.com>
	Jan Jancar <johny@neuromancer.sk>
	Marek Madejski <marekmadejski@yandex.com>
	Mark van Gelder <mvgmvgmvg@live.com>
	Peter Stahlecker <83544146+Peter0655@users.noreply.github.com>
	Peter Stahlecker <peter.stahlecker@gmail.com>
	Soumendra Ganguly <soumendraganguly@gmail.com>
	Zhuoyuan Li <zy.li@stu.pku.edu.cn>
```

Comment (Peter230655) [reply .mailmap]: So I delete line 1678 and then run bin/mailmap.py twice, correct?
Sorry about this mess. All I suggested to change were two words in a comment of KanesMethod - and it takes 1000 words for peopel to help me.

Comment (oscarbenjamin) [reply .mailmap]: The line added in .mailmap needs to be exactly the one that I showed. The name is not Peter230655 in the commits.

Comment (Peter230655) [reply .mailmap]: > The line added in .mailmap needs to be exactly the one that I showed. The name is not Peter230655 in the commits.

At least im my local ,mailmap it looks like this:
image
is this not correct?

Comment (Peter230655) [reply .mailmap]: is the space between ....com> and Peter230655 important?
Thanks!

Comment (oscarbenjamin) [reply .mailmap]: It should be exactly this:
```
Peter Stahlecker <peter.stahlecker@gmail.com> Peter Stahlecker <83544146+Peter230655@users.noreply.github.com>
```

**Comment by Peter230655** [comment]
Seems it finally passed all required tests?
**BIG** thanks to Jason and to Oscar for their patience with an old and clumsy guy!!

**Comment by moorepants** [comment]
Congrats on your first commit to SymPy!

**Comment by oscarbenjamin** [comment]
In future the .mailmap file will not need updating unless you change your name or email address in the git settings so it should be easier to open a new PR next time.

**Comment by Peter230655** [comment]
> Congrats on your first commit to SymPy!

Thanks! Without y'all's help I would NEVER have managed!
I am afraid, the next hurdle will be the tests, if I propose to change more than a word in a comment.
I read the docs - but as usual understood only maybe 10% :-(

I am glad I am done with .mailmap as I have no intention to change my name on my old age! :-)

**Comment by Peter230655** [comment]
I got this message, from Github, I suppose:
image
Why should I delete the branch and the fork, too?

**Comment by oscarbenjamin** [comment]
The branch is not needed any more so it can be deleted. The fork can be used again if you want to send another pull request. Sometimes someone might want to just send one pull request and then not send any others in future in which case the fork could be deleted as well.

**Comment by Peter230655** [comment]
> The branch is not needed any more so it can be deleted. The fork can be used again if you want to send another pull request. Sometimes someone might want to just send one pull request and then not send any others in future in which case the fork could be deleted as well.

I understand. What is good practice: deleting the branch or keep it?
If I delete it on GitHub, I also have to delete it locally, right?
**Thanks!**
(of course, I will keep the fork, I hope I can make contributions in future)

**Comment by moorepants** [comment]
Once the branch is merged into the main sympy repository you can delete the branch both locally on your computer and on your fork.

**Pull request merged**